src/lib/KeyStoreIndexeddb.js

```diff
@@ -203,7 +205,7 @@ class KeyStore {
      * stored the existing account labels. Both the 'accounts' database and cookie are
      * deleted afterwards.
      *
-     * @returns {Promise<boolean>}
+     * @returns {Promise<void>}
```

Comment:
Why change it?

Reply:
The method always returns `true`, so there is no need for a return value